eo-maven-plugin/src/test/java/integration/MojasIT.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2025 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package integration;
+
+import com.yegor256.Mktmp;
+import com.yegor256.MktmpResolver;
+import com.yegor256.WeAreOnline;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Map;
+import org.eolang.maven.AssembleMojo;
+import org.eolang.maven.ContainsFiles;
+import org.eolang.maven.FakeMaven;
+import org.eolang.maven.ParseMojo;
+import org.eolang.maven.PullMojo;
+import org.eolang.maven.ShakeMojo;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Integration tests for mojas.
+ *
+ * @since 0.1
+ */
+@SuppressWarnings({"JTCOP.RuleAllTestsHaveProductionClass", "JTCOP.RuleNotContainsTestWord"})
+@ExtendWith(WeAreOnline.class)
+@ExtendWith(MktmpResolver.class)
+final class MojasIT {
```

Comment:
@Suban05 In general you did the right thing, but it would be better to use `Farea` for integration tests. It will allow us not to make some classes and variables public. And it'll look more like integration test because you'll emulate the behaviour of mojo executed from pom.xml. The example of such test you can find [here](https://github.com/objectionary/eo/tree/master/eo-runtime/src/test/java/integration)